app/clients/sms/aws_pinpoint.py

```diff
@@ -83,13 +84,20 @@ def send_sms(
         reference,
         multi=True,
         sender=None,
+        created_at=datetime.utcnow(),
```

Comment:
we have a `created_at` column in the db for notifications and it is returned in the GET notifications response. if this somehow different than that? if so, should it have a different name? or is it possible to time this using the existing `created_at` field? is there any interaction between the existing field and this new property?

Reply:
That method doesn't have access to the notification object. Could have been `notification_created_at` but that seemed verbose.